pkg/generator/forms.go

```diff
@@ -229,6 +230,24 @@ func generateCompatibilityForm() Form {
 	}
 }
 
+func generateValidationForm() Form {
```

Comment:
WDYT about turning this into the "flags" form as the home for various flags in the future?

Reply:
This flag could alsoo potentially live in the compatibility form since it'll be force-enabled in v5.0.

Reply:
I thought it should exist in compatibility as well. I'll but it there